packages/shared/src/hooks/useSquadNavigation.ts

```diff
@@ -21,14 +22,14 @@ export const useSquadNavigation = (): UseSquadNavigation => {
         showLogin(AuthTriggers.CreateSquad);
         return;
       }
-      router.push(`/squads/new?origin=${props.origin}`);
+      router.push(`${webappUrl}/squads/new?origin=${props.origin}`);
```

Comment:
Would this not introduce double `//` in webapp since webappUrl is `/`?

Reply:
Not sure how `useRouter` handles it, but it works 😂 
Probably fixed under the hood.

https://github.com/dailydotdev/apps/assets/13744167/7a6aba0d-bb5b-4dad-af20-60c1e133ae79

Reply:
Yeah cool it works, but I would rather check those cases because it could just be a router bug and when they stop supporting that edge case our app breaks. 

Because now the string we are sending up there is `//squads/new?origin=sidebar` which is invalid.